src/main/java/seedu/address/logic/commands/task/AddTaskCommand.java

```diff
@@ -0,0 +1,65 @@
+package seedu.address.logic.commands.task;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_ADDRESS;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_DATE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_REMARK;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_WEIGHTAGE;
+
+import seedu.address.logic.commands.Command;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.task.Task;
+
+public class AddTaskCommand extends Command {
+
+    public static final String COMMAND_WORD = "add";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a task to TrackIt@NUS. "
+        + "Parameters: "
+        + PREFIX_NAME + "NAME "
+        + PREFIX_DATE + "DATE "
+        + PREFIX_ADDRESS + "ADDRESS "
+        + PREFIX_WEIGHTAGE + "WEIGHTAGE "
+        + PREFIX_REMARK + "REMARK\n"
+        + "Example: " + COMMAND_WORD + " "
+        + PREFIX_NAME + "CS2103T Final "
+        + PREFIX_DATE + "22/12/2020 "
```

Comment:
Ok. I suggest for now we leave it as is. Since Lesson and Task have a date component, we can standardise once both have been merged in and refactor it to be one of the 'commons'. Then that will be cleaner. Maybe leave for v1.3

Reply:
okay!